rct229/rulesets/ashrae9012019/section5/section5rule1.py

```diff
@@ -138,28 +138,28 @@ def get_calc_vals(self, context, data=None):
                     "$.output.output_instance", building_b270, False
                 )
 
-                # define a function to find the azimuth's corresponding key
-                find_key_for_azi = lambda azi: next(
-                    (
-                        key
-                        for key, range_ in azimuth_fen_area_dict_b.items()
-                        if int(key.split("-")[0]) <= azi <= int(key.split("-")[1])
-                    ),
-                    None,
-                )
-
-                azimuth_fen_area_dict_b = {
-                    f"{azi}-{azi+3}": ZERO.AREA for azi in range(3, 360, 3)
-                }
+                # define a function to get the azimuth's corresponding key
+                def get_key_for_azi(azi):
+                    azi_value = azi.to("degrees").magnitude
+                    low_bound = azi_value - (azi_value % 3)
+                    high_bound = low_bound + 3
```

Comment:
Good point.

I personally prefer to use the lambda function here as shown below because I believe the lambda function makes the code a bit cleaner.

```python
find_key_for_azi = lambda azi: next(
                    (
                        key
                        for key, _ in azimuth_fen_area_dict_b.items()
                        if int(key.split("-")[0])
                        <= azi.to(ureg("degree")).m
                        <= int(key.split("-")[1])
                    ),
                    None,
                )
```
I also corrected the missing 0-3 azimuth.

Reply:
@yunjoonjung-PNNL
I do not have much comment on your approach.
One key point is to keep in mind that we need lower bound to be inclusive and upper bound to be exclusive.
In the code, you need to update it to `< int(key.split("-")[1]`

Reply:
@yunjoonjung-PNNL @weilixu let me know if it is best to close this PR and move forward with https://github.com/pnnl/ruleset-checking-tool/pull/1257

Reply:
Great point! I addressed this part. I fixed the ```get_9012019_rmd_models``` function now and found that baseline_b90, 180, 270 still don't exist in the ```get_calc_vals``` method. I'll make this work and make a PR (not draft).

@JacksonJ-KC Thank you for doing this and since all the comments were addressed, I'll close this PR.